Make Dataset/script_dataset

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Non-obvious script name. May be something like process_reconstruction or convert_reconstruction would be better. Please think about it.